app/grandchallenge/annotations/validators.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import serializers
+from django.conf import settings
+
+
+def validate_grader_is_current_retina_user(grader, context):
+    """
+    This method checks if the passed grader equals the request.user that is passed in the context.
+    Only applies to users that are in the retina_graders group.
+    """
+    request = context.get("request")
+    if request and request.user.is_authenticated:
```

Comment:
Do you need a try block here to catch user lookups and convert those exceptions to `ValidationErrors`?

Reply:
Added a check for if `request.user is None` and clarified the method with an additional comment